src/js/techreport/tableLinked.js

```diff
@@ -21,18 +25,56 @@ class TableLinked {
       // Reset what's in the table before adding new content
       tbody.innerHTML = '';
 
+      const tableApps = content?.apps || this.pageFilters.app;
+
       // Collect the settings in an object
       const tableConfig = {
-        apps: this.pageFilters.app,
+        apps: tableApps,
         config: this.pageConfig?.[this.id]?.table,
         id: this.id,
       };
 
-      tableConfig.apps.forEach(app => {
-        const data = this.data[app];
+      const filters = new URLSearchParams(window.location.search);
+      const geo = filters.get('geo') || 'ALL';
+      const rank = filters.get('rank') || 'ALL';
+
+      // sort data
+      const sortEndpoint = component.dataset.sortEndpoint;
+      const sortMetric = component.dataset.sortMetric;
+      const sortKey = component.dataset.sortKey;
+      const client = component.dataset.client;
+
+      console.log(sortMetric);
+      if(sortMetric) {
+        this.dataArray = this.dataArray.sort((techA, techB) => {
+          // Sort techs by date to get the latest
+          const aSortedDate = techA.sort((a, b) => new Date(b.date) - new Date(a.date));
+          const bSortedDate = techB.sort((a, b) => new Date(b.date) - new Date(a.date));
+          const aLatest = aSortedDate[0];
+          const bLatest = bSortedDate[0];
```

Comment:
"Latest" should be the same for all technologies. It's possible for a technology to exist at one point in the dataset and then disappear forever, but this would always compare that ancient technology against the latest versions of all other modern technologies.

For example, October CMS shows up at the top of the top 1k CMS list with 361 origins:

https://technology-report-dot-httparchive.uk.r.appspot.com/reports/techreport/category?tech=&geo=ALL&rank=Top+1k&category=CMS

<img width="1204" alt="image" src="https://github.com/user-attachments/assets/0bcefa55-04f1-40d2-87dc-c5d22cb6a83a" />


But in the technology drilldown it's clearer that the most recent (and only) data point is from April 2022:

https://technology-report-dot-httparchive.uk.r.appspot.com/reports/techreport/tech?tech=October%20CMS&geo=ALL&rank=Top%201k

<img width="896" alt="image" src="https://github.com/user-attachments/assets/9b2faa97-a42e-4c91-b266-a97d214e309e" />

The expected behavior is for it to not appear on the Category page at all since it's not part of the latest dataset. That might also be a good reason to explicitly have a label like `Latest data: 2024-11-01` at the top of this report, similar to the others.